### PR TITLE
Cleanup jobs after modularisation

### PR DESCRIPTION
**Pull request by kolodziejczak**
**Description**

Changes proposed in this pull request:

- Cleanup jobs after modularisation

### CODE REVIEW

File: .github/actions/integration-tests/action.yaml [hunk 1]
```diff
@@ -28,15 +25,16 @@ runs:
...
-        sudo echo "127.0.0.1 local.kyma.dev" | sudo tee -a /etc/hosts
-        wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | sudo bash
-        [ -n "${KYMA_CLI_UNSTABLE}" ] && sudo curl -Lo /usr/bin/kyma https://storage.googleapis.com/kyma-cli-unstable/kyma-linux
-        [ -z "${KYMA_CLI_UNSTABLE}" ] && sudo curl -Lo kyma.tar.gz "https://github.com/kyma-project/cli/releases/latest/download/kyma_linux_x86_64.tar.gz" && sudo tar -C /usr/bin -zxvf kyma.tar.gz kyma
-        sudo chmod +x /usr/bin/kyma
-    - name: Provision Kyma and run tests
+    - name: Create Single Cluster
+      uses: AbsaOSS/k3d-action@4e8b3239042be1dc0aed6c5eb80c13b18200fc79 #v2.4.0
+      with:
+        cluster-name: "test-cluster-1"
```
Comment (werdes72) [inline .github/actions/integration-tests/action.yaml:31]: Why not `kyma-cluster` like it was before or maybe just `k3d-cluster`

File: .github/actions/upgrade-test/action.yaml [hunk 2]
```diff
@@ -32,18 +32,15 @@ runs:
...
-        [ -z "${KYMA_CLI_UNSTABLE}" ] && sudo curl -Lo kyma.tar.gz "https://github.com/kyma-project/cli/releases/latest/download/kyma_linux_x86_64.tar.gz" && sudo tar -C /usr/bin -zxvf kyma.tar.gz kyma
-        sudo chmod +x /usr/bin/kyma
-    - name: Provision k3d
-      shell: bash
-      run: |
-        sudo kyma provision k3d
+    - name: Create Single Cluster
+      uses: AbsaOSS/k3d-action@4e8b3239042be1dc0aed6c5eb80c13b18200fc79 #v2.4.0
+      with:
+        cluster-name: "test-cluster-1"
```
Comment (werdes72) [inline .github/actions/upgrade-test/action.yaml:38]: Same as above

**Review by werdes72 — CHANGES_REQUESTED** [review]

File: .github/actions/integration-tests/action.yaml (hunk 1 above)
Comment (kolodziejczak) [reply .github/actions/integration-tests/action.yaml:31]: I guess there is no reason for this name, taken straight from the other action

File: .github/actions/upgrade-test/action.yaml (hunk 2 above)
Comment (kolodziejczak) [reply .github/actions/upgrade-test/action.yaml:38]: I guess there is no reason for this name, taken straight from the other action

**Review by werdes72 — APPROVED** [review]

**Pull request merged**